soh/soh/Enhancements/gameplaystats.cpp

```diff
@@ -379,6 +379,22 @@ void SetupDisplayNames() {
     strcpy(timestampDisplayName[ITEM_SCALE_GOLDEN],     "Gold Scale:         ");
     strcpy(timestampDisplayName[ITEM_WALLET_ADULT],     "Adult's Wallet:     ");
     strcpy(timestampDisplayName[ITEM_WALLET_GIANT],     "Giant's Wallet:     ");
+    strcpy(timestampDisplayName[ITEM_WEIRD_EGG],        "Weird Egg:          ");
+    strcpy(timestampDisplayName[ITEM_GERUDO_CARD],      "Gerudo's Card:      ");
+    strcpy(timestampDisplayName[ITEM_COJIRO],           "Cojiro:             ");
+    strcpy(timestampDisplayName[ITEM_POCKET_EGG],       "Pocket Egg:         ");
+    strcpy(timestampDisplayName[ITEM_MASK_SKULL],       "Skull Mask:         ");
+    strcpy(timestampDisplayName[ITEM_MASK_SPOOKY],      "Spooky Mask:        ");
+    strcpy(timestampDisplayName[ITEM_MASK_KEATON],      "Keaton Mask:        ");
+    strcpy(timestampDisplayName[ITEM_MASK_BUNNY],       "Bunny Hood:         ");
+    strcpy(timestampDisplayName[ITEM_ODD_MUSHROOM],     "Odd Mushroom:       ");
+    strcpy(timestampDisplayName[ITEM_ODD_POTION],       "Odd Potion:         ");
+    strcpy(timestampDisplayName[ITEM_SAW],              "Poacher's Saw:      ");
+    strcpy(timestampDisplayName[ITEM_SWORD_BROKEN],     "Goron Sword (Broke):");
```

Comment:
hmm, the character limit here is kind of frustrating...

maybe just "Broken Sword", or actually, without parens "Broken Goron Sword" should fit

Reply:
```suggestion
    strcpy(timestampDisplayName[ITEM_SWORD_BROKEN],     "Broken Goron Sword: ");
```